src/index.html

```diff
@@ -11,7 +11,201 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```
3 point in checklist, I wrote about it in previous review, you need to fix it everywhere
![Знімок екрана 2022-09-27 о 17 44 52](https://user-images.githubusercontent.com/106310796/192558491-9115ef2d-c9d3-4367-8e3a-ee8fa345e6f8.png)

Reply:
why didn't you fixed this?